python/013-strstr.md

```diff
@@ -4,10 +4,13 @@
 - [next: 014. First Position of Target](014-first-position-of-target.md)
 
 ---
-
-put your own notes and solutions here.
-you can add any reference link such as [title](reference url) here.
-
+```
+class Solution:
+    def strStr(self, source, target):
+        if source is None or target is None:
+            return -1
+        return source.find(target)
```

Comment:
Although using library function could get an accepted result, I think it would be better for you to implement it from scratch (as the interviewer would also ask you to do so).